src/main/java/de/feu/massim22/group3/map/CellType.java

```diff
@@ -11,9 +11,6 @@ public enum CellType {
     DISPENSER_2,
     DISPENSER_3,
     DISPENSER_4,
-    DISPENSER_5,
```

Comment:
Theoretisch kann es glaube ich schon beliebig viele Blocktypen geben. Wenn man es genau machen wollte, müssten die verfügbaren Typen dynamisch sein. Aber das ist wahrscheinlich zu aufwendig, weil es sich auf so viele andere Stellen auswirkt und nicht notwendig.

Reply:
Ja ich hab da auch länger rumüberlegt wie ich es mache. Hatte erst als CellType DISPENSER und dann pro Zelle noch ein zusätzliches Attribut details. Das fand ich dann aber in der Map zu aufgeblasen. War aber sicher die sauberere Lösung.
Denke aber dass 5 Blocktypen ausreichen werden. Falls wir Probleme feststellen können wir die Anzahl noch hochschrauben.